.github/workflows/test.yaml

```diff
@@ -57,6 +57,11 @@ jobs:
                       allowed-to-fail: false
                       symfony-require: 6.3.*
                       variant: symfony/symfony:"6.3.*"
+                    - php-version: '8.2'
```

Comment:
I never worked a lot with devkit. But normally those files are generated automatically by devkit, or?

Reply:
Ah ok got it :+1: